UpdateData/README.md

```diff
@@ -1,2 +1,641 @@
-# sample-apps-nyu
-Sample apps project with NYU students
+# MongoDB Java Driver Update Data
+
+## Introduction
+
+This is a sample application that demonstrates MongoDB Java Driver's update functionality. It has four methods:
+
+* one that illustrates updating a single field
+* one that illustrates updating a single nested field
+* one that illustrates updating multiple fields using operator ($gt)
+* one that illustrates updating multiple nested fields
+
+## Getting Started
+
+The sample application demonstrates deletion of data from a MongoDB Atlas free-tier cluster. The cluster contains stores.orders collection which mimics an e-commerce database. The general schema is as follows: 
+
+```js
+{
+  _id: ObjectId(),
+  orderPlaced: datetime(),
+  total: NumberDecimal("153.00"),
+  subtotal: NumberDecimal("141.00"),
+  shipping: NumberDecimal("5.00"),
+  tax: NumberDecimal("7.00"),
+  status: ["shipped", datetime()],
+  shippingAddress: {
+    number: 345,
+    street: Alvin St.,
+    city: Madison
+    state: WI,
+    country: USA
+    postalCode: 53558
+  },
+  lineitems: [
+    { sku: "MDBTS001",
+      name: "Flannel T-shirt",
+      quantity: 10,
+      unit_price: NumberDecimal("9.00") },
+    { sku: "MDBTS002",
+      quantity: 5,
+      unit_price: NumberDecimal("10.00")}] 
+}
+```
+
+## Description 
+
+### Update One - UpdateOneOrder
+
+```java
+			Document newDocument = new Document();
+			newDocument.append("$set", new Document().append("subtotal", subtotal));
+
+			Document updateQuery = new Document().append("total", total);
+
+			collection.updateOne(updateQuery, newDocument);
+```
+
+This method illustrates updating one order. It takes *total* and *subtotal* as parameters and updates the first document with total equal to parameter, using the $set operator to update the subtotal field with parameter passed.
```

Comment:
Not too clear, so suggesting changes:
It takes *total* and *subtotal* as parameters and updates the first document where the total matches the parameter.  It uses the $set operator to update the subtotal field with the parameter passed.